src/templates/static/legal/metatrader/biel.jsx

```diff
@@ -109,13 +102,9 @@ const BIEL = () => (
             <li>{it.L('Trailing stops')}</li>
             <li>{it.L('Stop Loss Orders')}</li>
         </ol>
-        <p>{it.L('Before using the trading robots or Expert Advisors and forward trade, the client should test it on a demo account.')}</p>
         <p>{it.L('Actual trading results may not correspond to optimised or back-tested results.')}</p>
-        <p>{it.L('All software is to be used at the client\'s own risk. The Company will not be liable for any financial losses incurred using a third-party software. The Company is not associated with the development of the automated trading software or the Expert Advisors because they are exclusively developed and supported by third parties and not by the Company. The Company does not receive any form of financial and/or other benefits from permitting Expert Advisors to be used.')}</p>
```

Comment:
I don't think we should be removing the whole sentence. just the expert advisor. we need someone to confirm which parts exactly to remove, since to me the rest of the sentence seems important for automated trading. please confirm who needs to give us this information @aminroosta

Reply:
@negar-binary actually this is not referring to binary options EA specifically, it's generally referring to Expert Advisors from 3rd party developers.

It should stay as is (AFAICT), no need to change anything here.